File: ServiceApp/src/main/java/com/example/ServiceApp/dto/EquipmentDto.java
package com.example.ServiceApp.dto;

import lombok.*;

@AllArgsConstructor
@NoArgsConstructor
@Builder
@Getter
@Setter
public class EquipmentDto {

    private Long id;
    private String model;
    private String productCode;
    private Long producerId;
    private String producerName;
}

File: service-interface/src/app/dtos/equipmentDto.ts
import {ProducerDto} from "./producerDto";

export class EquipmentDto {
  id?: number;
  model?: string;
  productCode?: string;
  producerId?: number;
  producerName?: string;

  constructor(id: number, model: string, productCode: string, producerId: number, producerName: string) {
    this.id = id;
    this.model = model;
    this.productCode = productCode;
    this.producerId = producerId;
    this.producerName = producerName;
  }
}
